internal/services/formatters/generic/trivy/formatter.go

```diff
@@ -165,13 +165,21 @@ func (f *Formatter) addVulnerabilitiesOutput(vulnerabilities []*trivyVulnerabili
 	for _, vuln := range vulnerabilities {
 		addVuln := f.getVulnBase()
 		addVuln.Code = fmt.Sprintf("%s v%s", vuln.PkgName, vuln.InstalledVersion)
-		_, _, addVuln.Line = file.GetDependencyInfo(addVuln.Code, target)
+		addVuln.Line = f.getVulneLine(addVuln, target)
 		addVuln.File = target
 		addVuln.Details = vuln.getDetails()
 		addVuln.Severity = severities.GetSeverityByString(vuln.Severity)
 		addVuln.VulnHash = f.getOldHash(vuln.PkgName, *addVuln)
-		f.AddNewVulnerabilityIntoAnalysis(addVuln)
+		f.AddNewVulnerabilityIntoAnalysis(f.SetCommitAuthor(addVuln))
+	}
+}
+
+func (f *Formatter) getVulneLine(addVuln *vulnerability.Vulnerability, target string) string {
+	_, _, line, err := file.GetDependencyInfo(addVuln.Code, filepath.Join(f.GetConfigProjectPath(), target))
+	if err != nil {
+		logger.LogError(messages.MsgErrorGetDependencyInfo, err)
```

Comment:
Logging errors in the during the analysis will lead to a confusing output and will break the loading. Maybe we can log it only on debug log level and append this error into the  analysis struct to be printed in the end

Reply:
so we should do it to the functions in internal/utils/file.go too?

Reply:
Any log that is printed during the analysis will result in a broken loading and lost output errors in the middle of the output. So I believe it's better to return the error and print everything at the end, and for current `logger.LogError` to print only in debug mode.